test/classes/initializers/errors/errHandling/initWithTryBang2.chpl

```diff
@@ -0,0 +1,20 @@
+// Verifies we currently don't allow initializers to have try! statements in
+// their body.
+// We hope to eventually allow it.
+class Foo {
+  var x: int;
+
+  proc init() {
+    x = 10;
+    try! outerFunc();
+    super.init();
+  }
+}
+
+proc outerFunc() throws {
+  throw new Error();
+}
+
+var foo = new Foo();
```

Comment:
Similarly, this test generates this output:
```
uncaught Error:
   initWithTryBang2.chpl:15: thrown here
   initWithTryBang2.chpl:18: uncaught here
```

Reply:
Also looks OK to me.